### PR TITLE
Better format for patch command for INSTALL

### PR DESCRIPTION
**Pull request by gabriel-farache**
Better format for patch command for INSTALL for OCP related workflows

### CODE REVIEW

File: charts/request-vm-cnv/INSTALL.md [hunk 1]
```diff
@@ -71,11 +71,79 @@ You can patch the resource by running (update it if needed with your own values)
...
+            "name": "JIRA_USERNAME",
+            "value": "moti.asayag@gmail.com"
+          },
+          {
+            "name": "JIRA_API_TOKEN",
+            "value": "<token>"
+          },
+          {
+            "name": "OCP_API_SERVER_URL",
+            "value": "https://api.cluster-hz2vh.dynamic.redhatworkshops.io:6443"
```
Comment (rgolangh) [inline charts/request-vm-cnv/INSTALL.md:131]: before merging this, can you remove all those links with just example links and not real ones?

Comment (gabriel-farache) [reply charts/request-vm-cnv/INSTALL.md]: sure, done

**Comment by gabriel-farache** [comment]
@rgolangh why is this PR closed?

**Comment by gabriel-farache** [comment]
@rgolangh why was this PR closed?

**Pull request closed without merging**